### DISCUSSION TITLE
authelia bypassed after upgrade to v4.37.5

### DISCUSSION THREAD

**Discussion by ikkuranus** [authelia/authelia#4656] [Q&A]
I'm running Authelia docker on Unraid with SWAG as a reverse proxy to protect the admin portal of Vaultwarden and SWAG dashboard. It's also worth noting that I have MariaDB configured as the storage provider.

I don't know which version it was prior to the upgrade but it was using schema version 5.
Anyhow, after the upgrade it refused to start so I corrected all the configuration errors. At this point, it tried to start but failed to upgrade from schema version 5 to 7. I was unable to fix this so I ended up dropping all tables in the DB so it would rebuild.

Authelia was now started and the sites I previously protected are now available to all. I thought it might be due to the missing 2fa key so I went to auth.mydomain.com and reconfigured my totp keys for my account.

As a last ditch effort, I have commented all my rules out and set the default rule to two_factor (restarting as needed) but it still refuses to protect any of the sites that were previously protected.

This is my log as it's currently configured:
time="2022-12-26T22:11:24-06:00" level=info msg="Authelia v4.37.5 is starting"
time="2022-12-26T22:11:24-06:00" level=info msg="Log severity set to debug"

**Comment by james-d-elliott** [comment]
There are no requests being made from nginx to authelia as per the lack of logs.

**Reply by ikkuranus** [reply]
Any idea why that would be? Those sites (specifically the portions of them I configured) are completely inaccessible (internal server error) if I stop the authelia docker.
Here is my proxyconf for the vaultwarden docker https://pastebin.com/2Qt0fzSY  (note it's only configured to protect the /admin portal)

**Reply by james-d-elliott** [reply]
You'd need to look at the nginx error logs to find out why

**Reply by ikkuranus** [reply]
Apparently, I forgot to set
server:
  path: "authelia"

which was causing
2022/12/26 23:43:24 [error] 572#572: *114 authelia could not be resolved (3: Host not found), client: 2600:1700:8880:xxxx:xxxx:xxxx:xxxx:xxxx, server: host.*, request: "GET /admin HTTP/2.0", subrequest: "/authelia/api/verify", host: "host.domain.com"
2022/12/26 23:43:24 [error] 572#572: *114 auth request unexpected status: 502 while sending to client, client: 2600:1700:8880:xxxx:xxxx:xxxx:xxxx:xxxx, server: host.*, request: "GET /admin HTTP/2.0", host: "host.domain.com"

**Reply by james-d-elliott** [reply]
Host not found indicates the authelia container is not resolvable by nginx, maybe it's not on the correct network, or docker dns is not configured or working correctly.

**Reply by ikkuranus** [reply]
I have one more question not related to this issue. Can I ask it here or should I start a new thread?

**Reply by james-d-elliott** [reply]
Sure.

**Reply by ikkuranus** [reply]
The previous issue is solved but I'm now trying to use webauthn and my yubikey isn't recognised on subsequent login attempts after registering it. browser is ms edge chromium if that matters

**Reply by james-d-elliott** [reply]
This is probably because it's on a different origin URL, the default swag setup doesn't work for webauthn and we recommend people instead use the official integration guide and configure it on a subdomain rather than a subpath for every other subdomain: https://www.authelia.com/integration/proxies/swag/

**Reply by ikkuranus** [reply]
Ok, thank you I'll switch over to that when I get a chance.

**Reply by james-d-elliott** [reply]
There's specific mention of why here: https://www.authelia.com/integration/proxies/swag/#swag-caveat

In short the Webauthn standard doesn't allow it.
